imagemanager/imagemanager.go

```diff
@@ -0,0 +1,1084 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
there is some file imagemanager/--no-auto-compress  in this commit, Remove it

Reply:
Done